libs/process_limits/Unit_memory_limit.ml

```diff
@@ -23,22 +23,27 @@ let with_debug_alarm f =
 (*
    Grow the stack until some limit expressed in bytes.
```

Comment:
I think we can just delete this whole test now? Since the stack is no longer real